Sources/App/Features/Slots/Models/Slot.swift

```diff
@@ -19,8 +19,13 @@ final class Slot: Codable, Model, Content, @unchecked Sendable {
     @Field(key: "duration")
     var duration: Double
 
-    @Parent(key: "event_id")
-    var event: Event
+    // DO NOT USE (June 2024)
+    // This will be removed in a future PR as part of a cleanup - it needs to be done this way for safe migrations.
```

Comment:
⚠️  There is a very complex sequence of events at play here, especially with the migrations.

If I delete the Event ID as part of this PR, then any migration which relies on the Event ID (including the EventDay migration) will then not work. Instead, I've opted to maintain both systems temporarily but have worked to remove all production uses of the property.

This means while the data is there, and is being populated, we've minimised use of it and we can remove it trivially in a future PR (after everybody has migrated).